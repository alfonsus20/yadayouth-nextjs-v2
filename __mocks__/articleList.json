{
    "count": 3,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 5,
            "title": "YADA IMPACT 3.0: Digital Era Empowers the Child Potential",
            "author": null,
            "timePublised": "2022-03-29T12:47:13Z",
            "articleImage": "https://i.ibb.co/VtjP8s6/Untitled-design.png",
            "topic": "Webinar",
            "preview": "On Sunday (13/03), Yada Youth Indonesia successfully held its 3rd Yada Impact titled “Digital Era Empowers the Child Potential”.",
            "content": "<p dir=\"ltr\" style=\"line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;\"><span style=\"font-family: &quot;Helvetica Neue&quot;; font-size: 12px;\">﻿</span><span style=\"font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;\">On Sunday (13/03), Yada Youth Indonesia successfully held its 3rd Yada Impact titled “Digital Era Empowers the Child Potential”. The event was open to the public via Zoom Cloud Meeting, featuring two incredible speakers, Muflih Dwi Fikri (CEO of Global Millennial Group) and Angel Nathania Devi, S.Psi., M.Psi. (Clinical Child and Adolescent Psychologist at Klinik Jiwa Selaras and RS Sumber Kasih). Each speaker presented the sub themes of Yada Impact 3.0 which is “Be Smart with Digital Literacy” by Muflih and “Explore the Child’s Potential, not Burden the Child” by Devi.&nbsp;</span></p><p><span id=\"docs-internal-guid-2653db64-7fff-b028-cee1-93fdbf69b04c\"></span></p><p dir=\"ltr\" style=\"line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;\"><span style=\"font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;\"><br></span></p><p dir=\"ltr\" style=\"line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;\"><span style=\"font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;\">Muflih opened his speech by explaining that the generation has developed rapidly, especially with technology that has become much easier to use. He also shared his experience in Global Millennial Group while emphasizing the importance of having digital literacy in the present era. “Time will always move on, it wont stop and wait for anyone. That’s why we have to always upgrade ourselves to catch up with the era,” say Muflih. Moving on to the next speaker, Devi shared that parental pressure can cause psychological burden to the child. “Parents experience excessive anxiety and are ambitious, controlling and rigid, and forcing their children to learn beyond their limits. It makes learning become a structured and controlled activity by parents that causes children to experience mental problems such as anxiety, frustration, and depression,” Devi explained. “As parents, there are several things we can do, such as supporting their interest, teaching them to set boundaries and take responsibility, and never compare them to other children. Appreciate their effort, not only their achievements,”</span></p>"
        },
        {
            "id": 4,
            "title": "Dampak Kemiskinan pada Akses Anak ke Edukasi Seks",
            "author": "Fishya Elvin",
            "timePublised": "2021-11-11T06:00:00Z",
            "articleImage": "https://i.ibb.co/MMQmLkg/Hand-writing-with-chalk-sex-education-concept-on-blackboard.jpg",
            "topic": "Education",
            "preview": "Kemiskinan menyebabkan sulitnya seseorang mengakses pendidikan, tanpa terkecuali pendidikan seks yang sangat dibutuhkan anak usia dini untuk meningkatkan kesadaran mereka terhadap tubuh mereka sendiri.",
            "content": "<p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Menjadi agenda dalam salah satu Sustainable Development Goals dari United Nations, mengurangi angka kemiskinan telah menjadi tujuan penting bagi pemerintah saat ini di seluruh belahan dunia. Kemiskinan telah menjadi permasalahan kompleks dan merupakan akar dari kasus kelaparan, pengangguran, kesehatan, sampai pendidikan. Berkenaan dengan hal ini, semua lapisan masyarakat juga terkena dampaknya, tanpa terkecuali anak-anak.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Kemiskinan berdampak besar bagi perkembangan anak, terutama dalam segi pendidikan. Tingginya biaya pendidikan membuat orang tua kesulitan dalam memenuhi kebutuhan seorang anak. Diketahui dalam penelitian </span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Education and Poverty, Journal of Human Development</span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\"> (Tilak, 2002) bahwa tingkat pendidikan yang rendah pada suatu daerah disebabkan oleh rendahnya pendapatan masyarakat sehingga mereka tidak mampu meraih peluang ekonomi. Hal ini berujung pada sebagian anak yang terpaksa harus putus sekolah dan menjadi tulang punggung keluarga.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Rendahnya pendidikan ini membuat anak memiliki pengetahuan yang minim dan berakhir melakukan segala cara untuk memenuhi rasa keingintahuannya itu. Di tengah perkembangan teknologi yang semakin canggih dan mudah diakses oleh siapapun, anak berakhir mencari tahu segala hal yang ingin diketahuinya secara mandiri melalui internet. Tidak peduli cara yang ditempuh benar atau salah, rasa penasaran dan keingintahuan bisa menyesatkannya.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Tidak heran kita banyak menemukan beberapa kasus kekerasan seksual yang melibatkan seorang anak baik sebagai pelaku maupun korbannya. Belajar dari internet, menonton film dewasa, membuat mereka sulit untuk membedakan mana yang benar dan mana yang salah, sehingga mereka seringkali meniru adegan dewasa dalam film. Akhirnya, anak pada usia dini mengalami kejadian yang tidak diinginkan, seperti kecanduan menonton video dewasa, pergaulan bebas, sampai hamil di luar nikah.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Selain itu, ketidaktahuan seorang anak akan area dalam tubuh mereka yang tidak boleh disentuh oleh siapapun menyebabkan seorang anak rentan menjadi korban kekerasan seksual. Bahkan pada beberapa kasus, anak tersebut tidak menyadari bahwa selama ini telah menjadi korban dari kasus kekerasan seksual yang dilakukan oleh orang dewasa.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Hal tersebut membuktikan bahwa kemiskinan menyebabkan sulitnya seseorang mengakses pendidikan, tanpa terkecuali pendidikan seks yang sangat dibutuhkan anak usia dini untuk meningkatkan kesadaran mereka terhadap tubuh mereka sendiri.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Kemiskinan yang mengakibatkan rendahnya pendidikan seorang anak tersebut memang dapat diberantas dengan menegakkan kebijakan pemerintah yang kurang memperhatikan pendidikan tinggi bagi masyarakat yang kurang mampu. Dengan begitu, seorang anak akan mendapatkan pendidikan yang layak bahkan lebih baik. Namun, terdapat alternatif lain yang dapat diterapkan oleh orang tua dalam mendidik dan memberikan edukasi pada anak.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Edukasi seks dari lingkungan terdekat, seperti keluarga ataupun orang tua juga harus digencarkan. Tidak ada istilah ‘terlalu cepat’ dalam memberikan edukasi seks kepada anak. Memberikan pemahaman sedini mungkin akan membantu seorang anak memahami seks dan seksualitas yang berkaitan dengan tubuh dan kesehatan mereka. Berhubungan juga dengan bagaimana seorang anak mengembangkan tubuhnya, memahami dan mampu mengekspresikan perasaan intim atau ketertarikan pada seseorang, dan bagaimana anak dapat menjalani hubungan yang baik dengan lawan jenisnya.</span></p><p><span id=\"docs-internal-guid-790bcf5e-7fff-60e2-b674-f12d3ac7c4db\"></span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Mengajarkan kesadaran pada tubuh sendiri, seperti contoh kasus yang telah dipaparkan sebelumnya, menyimpulkan bahwa seorang anak harus tahu sejak dini terkait area intim yang tidak boleh disentuh orang lain, sehingga mereka tahu kapan harus berkata tidak dan kapan harus berkata iya. Membicarakan hal ini dapat menjaga keamanan anak, agar terlindung dari kekerasan seksual yang seringkali memanfaatkan sifat ketidaktahuan seorang anak yang masih belia. Kita tidak boleh membiarkan rendahnya tingkat pendidikan dan kekerasan seksual menghancurkan masa depan seorang anak.</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Penulis:</span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\"> Fishya Elvin</span></p><p dir=\"ltr\" style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Referensi:</span></p><ul><li style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\"><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">True Relationships and Reproductive Health (2018). </span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Communicating about sexuality with children</span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">. Brisbane: True.</span></li><li style=\"line-height: 1.38; margin-top: 12pt; margin-bottom: 12pt;\">Tilak, J. B. G. (2002). Education and Poverty. <span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">Journal of Human Development</span><span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;\">, 3(2).</span></li></ul>"
        },
        {
            "id": 3,
            "title": "Losing 10 Kgs a Week, Is It Possible? – Unhealthy Eating Habits of Youths",
            "author": "Clarissa",
            "timePublised": "2021-05-09T05:28:10Z",
            "articleImage": "https://images.unsplash.com/photo-1614505255161-ad10bb0e1a4e",
            "topic": "Lifestyle",
            "preview": "The digital era has allowed us to search for almost everything on the Internet. However, how much of this information can you totally rely on, especially when it comes to your health?",
            "content": "<p>The digital era has allowed us to search for almost everything on the Internet. However, how much of this information can you totally rely on, especially when it comes to your health? According to Paediatrics Child Health, concerns regarding weight and body shape is extremely common during adolescent years. In addition, youths are continuously being exposed to the health risks of obesity and poor nutrition, plus the unrealistic thin beauty ideal standards portrayed on social media by celebrities and other public figures.</p><p><br></p><p>Due to media exposure, you might have seen a number of weight loss plans online. A popular example would be the Keto diet, or also known as the low-carb eating plan. With loads of success stories on social media, this dieting plan has been extremely prevalent among the public, with many of them completely eliminating rice or certain fruits from their meals to achieve their ‘ideal look’. However, many still do not understand the way it works or its drawbacks.</p><p><br></p><p>Consequently, it is important to understand how you can reach your goals without following bad diets and starving yourself. Based on research, top dietitians and nutritionists have recommended the following strategies that can actually help you to see results:</p><p><br></p><h3><b><span style=\"font-size: 24px;\">CONTROL your Calorie Intake &amp; DO NOT eliminate certain foods!</span></b></h3><p>You probably would have seen people around you totally cutting out carbs and sugar when trying to lose weight. However, registered dietitian nutritionist, Tara Collingwood suggests that dieting is all about your Calorie Balance! Even healthy foods can make you gain weight if you do not control your calorie intake (eg. Avocados &amp; Peanuts are high in calories). Just like how one salad does not instantly make you lose weight, a bowl of creamy pasta will not make you gain weight either.</p><h3 style=\"text-align: left;\"><br><b><span style=\"font-size: 24px;\">GO on weekly cheat meals!</span></b></h3><p style=\"text-align: left;\">When you get obsessed with a strict meal plan, you are more likely to give up and feast on all your cravings after sometime. This is why registered dietitian, Kristin Reisinger recommends going on a weekly indulgence. In fact, studies have shown that a large cheat meal helps you reach your ‘ideal’ weight faster by maintaining your metabolism or preventing you from reaching a ‘weight loss plateau (a condition when your body adapts to the continuous calorie deficit)’. After a cheat meal, it is totally normal to gain weight, due to factors such as water retention or bloating, and NOT FAT. Do not panic, all you have to do is to get back to a balanced diet ASAP, and you will return to your initial weight very soon.</p><h3 style=\"text-align: left;\"><br><b><span style=\"font-size: 24px;\">DO NOT compare yourself with others!</span></b></h3><p style=\"text-align: left;\">Just because your friend loses weight by eating 1700 Calories a day, doesn’t mean that it will do the same to you. Registered dietitian nutritionist, Caroline Passerrello stated that “everyone has different nutritional needs, so everyone’s ideal diet will be different”. Instead of obsessing over calories, focus on how you feel and eat in response to physiological hunger, and stop when you feel that you have eaten enough.</p><p style=\"text-align: left;\"><br></p><h3 style=\"text-align: left;\"><b><span style=\"font-size: 24px;\">What Should I Eat?</span></b></h3><p>Studies have proven that <b>Proteins</b> (eg. hard-boiled eggs, oatmeal) helps to curb your appetite and keep you full for longer hours. Also, do not forget your plate of <b>Vegetables</b>! Not only are they low in calories, but they also make excellent sources of fiber and water as well, which keeps you satisfied. Lastly, remember that <b>Snacking</b> is totally okay! You can always search for low-calorie and filling snacks, or just grab a small portion of your favourite cake and enjoy it.</p><p><br></p><p>Above all, the key to sustainable weight loss is to develop lifestyle changes that you can keep up with in the long term. Adding exercise to your day, switching high-calorie desserts for fruits after a huge meal, and being more mindful of your portions are some of the changes that you can easily put into daily practice. Take your time to reflect on what you enjoy most in terms of exercise or write down a list of balanced meals you actually find tasty. Then, all you have to do is to be patient, stay consistent, and enjoy the process!</p><p><br></p><p>Sources:<br></p><ul><li>Paediatrics Child Health</li><li>Tara Collingwood (Registered Dietitian)</li><li>Amy Gorin (Registered Dietitian)</li><li>Kristin Reisinger (Registered Dietitian and Trainer)</li><li>Medical News Today</li></ul><div><br></div>"
        }
    ]
}