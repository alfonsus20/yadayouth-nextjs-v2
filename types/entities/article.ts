import { Paginated } from "../apiResponse";

export type Article = {
  id: number;
  title: string;
  author: string | null;
  timePublised: string;
  articleImage: string;
  topic: string;
  preview: string;
  content: string;
};

export type ArticleDto = {
  previous: number | null;
  next: number | null;
  result: Article;
};

export type ArticleList = Paginated<Article>;
