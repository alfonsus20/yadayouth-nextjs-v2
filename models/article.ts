import { AxiosPromise } from "axios";
import api from "../api";
import { APIResponsePaginated } from "../types/apiResponse";
import { Article } from "../types/entities/article";

export const fetchArticles = (): APIResponsePaginated<Article> => {
  return api.get("/article");
};

export const fetchArticlesById = (id : string): AxiosPromise<Article> => {
  return api.get(`/article/${id}`);
};
