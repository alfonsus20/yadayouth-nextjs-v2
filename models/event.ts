import api from "../api";
import { APIResponsePaginated } from "../types/apiResponse";
import { Event } from "../types/entities/event";

export const fetchPastEvents = (): APIResponsePaginated<Event> => {
  return api.get("/past-event");
};
