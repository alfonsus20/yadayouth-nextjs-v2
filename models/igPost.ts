import { igApi } from "../api";
import { APIResponse } from "../types/apiResponse";
import { IGPost } from "../types/entities/igPost";

export const fetchIGPosts = (): APIResponse<Array<IGPost>> => {
  return igApi.get("/");
};
