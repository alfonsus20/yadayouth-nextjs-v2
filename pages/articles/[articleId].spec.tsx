import ArticleDetail, { getStaticPaths, getStaticProps } from './[articleId].page';
import * as svc from '../../models/article';
import articleList from '../../__mocks__/articleList.json';
import articleSingle from '../../__mocks__/articleSingle.json';
import { fireEvent, render, screen } from '@testing-library/react';
import mockRouter from 'next-router-mock';

jest.mock('next/router', () => require('next-router-mock'));

describe('Article detail unit test', () => {
  beforeEach(() => {
    jest.spyOn(svc, 'fetchArticles').mockResolvedValue({
      data: articleList,
      status: 200,
      statusText: 'Ok',
      headers: {},
      config: {},
    });
  });

  afterEach(() => {
    jest.clearAllMocks();
  });

  it('should match snapshot', async () => {
    jest.spyOn(svc, 'fetchArticlesById').mockResolvedValue({
      data: articleSingle,
      status: 200,
      statusText: 'Ok',
      headers: {},
      config: {},
    });

    await getStaticProps({ params: { articleId: '5' } });
    await getStaticPaths({});

    const container = render(
      <ArticleDetail
        data={articleSingle.result}
        next={articleList.next}
        prev={articleList.previous}
      />
    );
    expect(container).toMatchSnapshot();
  });

  it('should handle not found article', async () => {
    jest.spyOn(svc, 'fetchArticlesById').mockRejectedValue({
      data: null,
      status: 404,
      headers: {},
      config: {},
    });

    await getStaticProps({ params: { articleId: '5' } });
    const res = await getStaticProps({ params: { articleId: '12' } });
    expect(res).toEqual({ props: {}, notFound: true });
  });

  it('should show prev button when prev prop exists and redirect to correct article detail page', async () => {
    jest.spyOn(svc, 'fetchArticlesById').mockResolvedValue({
      data: articleSingle,
      status: 200,
      statusText: 'Ok',
      headers: {},
      config: {},
    });

    await getStaticProps({ params: { articleId: '5' } });

    render(<ArticleDetail data={articleSingle.result} prev={4} next={null} />);

    expect(screen.getByTestId('btn-prev')).toBeVisible();

    fireEvent.click(screen.getByTestId('btn-prev'));
    expect(mockRouter.pathname).toEqual('/articles/4');
  });

  it('should show next button when next prop exists and redirect to correct article detail page', async () => {
    jest.spyOn(svc, 'fetchArticlesById').mockResolvedValue({
      data: articleSingle,
      status: 200,
      statusText: 'Ok',
      headers: {},
      config: {},
    });

    await getStaticProps({ params: { articleId: '5' } });

    render(<ArticleDetail data={articleSingle.result} next={6} prev={null} />);

    expect(screen.getByTestId('btn-next')).toBeVisible();

    fireEvent.click(screen.getByTestId('btn-next'));
    expect(mockRouter.pathname).toEqual('/articles/6');
  });
});
