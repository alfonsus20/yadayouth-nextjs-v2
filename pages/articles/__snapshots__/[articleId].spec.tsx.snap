// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article detail unit test should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="max-w-screen-xl mx-auto pt-8 px-8 lg:px-0"
      >
        <div
          class="mb-4"
        >
          <h2
            class="text-gray-600 font-bold text-3xl mb-2"
          >
            YADA IMPACT 3.0: Digital Era Empowers the Child Potential
          </h2>
          <p
            class="text-orange-500"
          >
            Tuesday, 29 March 2022
          </p>
        </div>
        <div
          class="flex justify-center mb-4"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27900%27%20height=%27480%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="gambar"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover; object-position: center;"
            />
            <noscript />
          </span>
        </div>
        <div>
          <p
            dir="ltr"
            style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
          >
            <span
              style="font-family: \\"Helvetica Neue\\"; font-size: 12px;"
            >
              ﻿
            </span>
            <span
              style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
            >
              On Sunday (13/03), Yada Youth Indonesia successfully held its 3rd Yada Impact titled “Digital Era Empowers the Child Potential”. The event was open to the public via Zoom Cloud Meeting, featuring two incredible speakers, Muflih Dwi Fikri (CEO of Global Millennial Group) and Angel Nathania Devi, S.Psi., M.Psi. (Clinical Child and Adolescent Psychologist at Klinik Jiwa Selaras and RS Sumber Kasih). Each speaker presented the sub themes of Yada Impact 3.0 which is “Be Smart with Digital Literacy” by Muflih and “Explore the Child’s Potential, not Burden the Child” by Devi. 
            </span>
          </p>
          <p>
            <span
              id="docs-internal-guid-2653db64-7fff-b028-cee1-93fdbf69b04c"
            />
          </p>
          <p
            dir="ltr"
            style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
          >
            <span
              style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
            >
              <br />
            </span>
          </p>
          <p
            dir="ltr"
            style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
          >
            <span
              style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
            >
              Muflih opened his speech by explaining that the generation has developed rapidly, especially with technology that has become much easier to use. He also shared his experience in Global Millennial Group while emphasizing the importance of having digital literacy in the present era. “Time will always move on, it wont stop and wait for anyone. That’s why we have to always upgrade ourselves to catch up with the era,” say Muflih. Moving on to the next speaker, Devi shared that parental pressure can cause psychological burden to the child. “Parents experience excessive anxiety and are ambitious, controlling and rigid, and forcing their children to learn beyond their limits. It makes learning become a structured and controlled activity by parents that causes children to experience mental problems such as anxiety, frustration, and depression,” Devi explained. “As parents, there are several things we can do, such as supporting their interest, teaching them to set boundaries and take responsibility, and never compare them to other children. Appreciate their effort, not only their achievements,”
            </span>
          </p>
        </div>
        <div
          class="flex"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="max-w-screen-xl mx-auto pt-8 px-8 lg:px-0"
    >
      <div
        class="mb-4"
      >
        <h2
          class="text-gray-600 font-bold text-3xl mb-2"
        >
          YADA IMPACT 3.0: Digital Era Empowers the Child Potential
        </h2>
        <p
          class="text-orange-500"
        >
          Tuesday, 29 March 2022
        </p>
      </div>
      <div
        class="flex justify-center mb-4"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27900%27%20height=%27480%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="gambar"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover; object-position: center;"
          />
          <noscript />
        </span>
      </div>
      <div>
        <p
          dir="ltr"
          style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
        >
          <span
            style="font-family: \\"Helvetica Neue\\"; font-size: 12px;"
          >
            ﻿
          </span>
          <span
            style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
          >
            On Sunday (13/03), Yada Youth Indonesia successfully held its 3rd Yada Impact titled “Digital Era Empowers the Child Potential”. The event was open to the public via Zoom Cloud Meeting, featuring two incredible speakers, Muflih Dwi Fikri (CEO of Global Millennial Group) and Angel Nathania Devi, S.Psi., M.Psi. (Clinical Child and Adolescent Psychologist at Klinik Jiwa Selaras and RS Sumber Kasih). Each speaker presented the sub themes of Yada Impact 3.0 which is “Be Smart with Digital Literacy” by Muflih and “Explore the Child’s Potential, not Burden the Child” by Devi. 
          </span>
        </p>
        <p>
          <span
            id="docs-internal-guid-2653db64-7fff-b028-cee1-93fdbf69b04c"
          />
        </p>
        <p
          dir="ltr"
          style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
        >
          <span
            style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
          >
            <br />
          </span>
        </p>
        <p
          dir="ltr"
          style="line-height:1.7999999999999998;text-align: justify;margin-top:0pt;margin-bottom:0pt;"
        >
          <span
            style="font-size:12pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"
          >
            Muflih opened his speech by explaining that the generation has developed rapidly, especially with technology that has become much easier to use. He also shared his experience in Global Millennial Group while emphasizing the importance of having digital literacy in the present era. “Time will always move on, it wont stop and wait for anyone. That’s why we have to always upgrade ourselves to catch up with the era,” say Muflih. Moving on to the next speaker, Devi shared that parental pressure can cause psychological burden to the child. “Parents experience excessive anxiety and are ambitious, controlling and rigid, and forcing their children to learn beyond their limits. It makes learning become a structured and controlled activity by parents that causes children to experience mental problems such as anxiety, frustration, and depression,” Devi explained. “As parents, there are several things we can do, such as supporting their interest, teaching them to set boundaries and take responsibility, and never compare them to other children. Appreciate their effort, not only their achievements,”
          </span>
        </p>
      </div>
      <div
        class="flex"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
